src/wallet/wallet.cpp

```diff
@@ -3135,6 +3130,11 @@ bool CWallet::CreateTransaction(interfaces::Chain::Lock& locked_chain, const std
         }
     }
 
+    if (nFeeRet > this->m_default_max_tx_fee) {
```

Comment:
style-nit:

```suggestion
    if (nFeeRet > m_default_max_tx_fee) {
```

`m_` already means `this->`, no need for duplicate.